### PR TITLE
Jekyll Community

### PR DESCRIPTION
**Pull request by chrisfinazzo**
See #5057 for discussion related to this PR.

### CODE REVIEW

**Comment by pathawks** [comment]
Why not just force push this to the existing PR (#5095)?

**Comment by chrisfinazzo** [comment]
Tl;dr Force pushing is [gross][], so I try to avoid it when possible. See @cauerego's comment in the link for a more nuanced explanation.

[gross]: http://stackoverflow.com/questions/5509543/how-do-i-properly-force-a-git-push

**Comment by pathawks** [comment]
> Force pushing is gross

One conversation that spans across several PRs is gross, but I don't mean to derail things here. Carry on.

**Comment by chrisfinazzo** [comment]
@pathawks https://media0.giphy.com/media/l2JJJHGbjqUDZJp8A/200w.gif

File: site/_data/docs.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@
   - troubleshooting
   - sites
   - resources
+  - jekyllconf
```
Comment (parkr) [inline site/_data/docs.yml]: This needs to be removed :smile:

**Comment by chrisfinazzo** [comment]
Ack! Missed that, will update.

**Comment by cauerego** [comment]
Going on the tangent for which I was mentioned here and without knowing if this is gross or not... Nothing against force pushes @parkr ?! Why not?

**Comment by chrisfinazzo** [comment]
@cauerego, As you described, with great power comes great responsibility. For that way lies madness 😏

**Comment by cauerego** [comment]
Yikes, now I'm completely lost @chrisfinazzo! I was also probably not clear: I'm just hoping Mr Moore could make me see a completely different angle from what I brought on the link you've [pointed](http://stackoverflow.com/a/16702355/274502) and in which I don't recall bringing up Mr Parker. 😄

**Comment by chrisfinazzo** [comment]
@cauerego As you indicated in the SO comment I linked, rewriting published history is strongly discouraged. It requires people to scary things in order to incorporate new changes and is generally not a situation you want to find yourself in.

Even if it's only in a branch, force-pushing has that smell to it 👎🏻

**Comment by parkr** [comment]
Did you check this on mobile? We might need to add a link to this page in the mobile nav.

**Comment by envygeeks** [comment]
@cauerego force pushes matter little on a development branch, that's what temporary development branches are for, forcing, cleaning and organizing a patch to send.  In this case, it's just automatically done for you through Github.

**Comment by parkr** [comment]
Took a look locally. All is well. 👍

**Comment by parkr** [comment]
@jekyllbot: merge +site

**Comment by chrisfinazzo** [comment]
@parkr Yeah, looked at it with the iOS Simulator and everything seems to be in order. I am a bit wary of the padding in portrait orientation, but that's not a huge deal.
screen shot 2016-07-12 at 10 59 19 pm

**Pull request merged**